evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -439,7 +442,7 @@ def parse_sdescs_and_recogs(
                 (re.search(rquery, text, _RE_FLAGS), obj, text) for obj, text in candidate_map
             )
             # filter out any non-matching candidates
-            bestmatches = [(obj, match.group()) for match, obj, text in matches if match]
+            bestmatches = [(obj, m.group()) for m, obj, text in matches if m]
```

Comment:
Why turn `match` into a one-letter variable `m`? PEP8 clearly recommends staying >1 character length for all Python variables.

Reply:
Ah, my bad, I was mimicking the format I've seen used for list comprehensions etc. elsewhere in the code.

As for why, it's because `match` is a builtin keyword now and I get paranoid about overwriting those. >.> So I can just take this change out.

Reply:
I shouldn't be using one-letter vars in list comprehensions, but it's possible it has snuck in somewhere (there's a lot of code after all). But it's actually a fair point concerning the use of `match`. We shouldn't use an existing python keyword. But one can still use something other than a one-letter one to appeal to PEP8 :)

Reply:
Sounds good! I'll do something like mat or mtch instead.